src/main/java/redis/clients/jedis/util/Version.java

```diff
@@ -0,0 +1,40 @@
+package redis.clients.jedis.util;
+
+import java.util.Arrays;
+
+public class Version implements Comparable<Version> {
+
+    private String version;
+    private Integer[] numbers;
+
+    public Version(String version) {
+        if (version == null) throw new IllegalArgumentException("Version can not be null");
+        this.version = version;
+        this.numbers = Arrays.stream(version.split("\\.")).map(n -> Integer.parseInt(n)).toArray(Integer[]::new);
```

Comment:
We're doing null checks but there can be many other checks; i.e. valid format, valid values, etc.

Reply:
Right, we might choose to introduce bunch of checks besides corresponding excpetions/messages etc, this one is in the way to take it more pragmatic and put required function in place as i mentioned in the other comment. And have no further intention to grow a generic 'Version' implementation to handle all at once.